src/app/components/NetworkSwitcher/index.js

```diff
@@ -15,6 +15,8 @@ import * as Neoscan from '../../utils/api/neoscan'
 
 import { truncateString } from '../../utils/api/neon'
 
+import * as _ from 'lodash'
```

Comment:
you could also do the following import { isArray } from "lodash" 
this way you don't import the whole library if not needed